src/kalman.js

```diff
@@ -60,6 +60,23 @@ export default class KalmanFilter {
     return this.x;
   }
 
+  /**
+  * Predict next value
+  * @param  {Number} [u] Control
+  * @return {Number}
+  */
+  predict(u = 0) {
+    return (this.A * this.x) + (this.B * u);
+  }
+  
+  /**
+  * Predict next covariance
+  * @return {Number}
+  */
+  predictCovariance() {
```

Comment:
Could you change the docblock and the name of this function? The covariance here describes the uncertainty of the prediction, so we are not really 'predicting' the covariance. To keep it in style of the other (simply named) functions, 'uncertainty()' would probably be clear enough.

Reply:
That sounds like a great idea!